lib/embed/leaderboard_embed.py

```diff
@@ -4,6 +4,14 @@
 color_list = [Assets.easy, Assets.medium, Assets.hard]
 medal_list = ["🥇", "🥈", "🥉"]
 
+role_emojies = {
+    "1280167940775481365": Assets.student, # llclass
+    "1085444843691577404": Assets.leader, # leader
+    "1148541475353788477": Assets.blob_victory, # lotm
+    "1087746207511757002": Assets.cat_typing, # bot team
+    "1085445066484621362": ":military_medal:", # core team
+}
```

Comment:
I think this is a good opportunity to consider starting refactoring roles/channel/etc IDs to an Enum/Class with static attributes (similar to `Assets` in `utils.asset` which stores emojis). I wasn't always fond of "magic IDs" scattered around in the codebase.

What do you think? Is it worth the effort? Is there any possible dealbreaker scenarios that the change will introduce?